### DISCUSSION TITLE
Is there any chance to import/sync my office 365 calendar?

### DISCUSSION THREAD

**Discussion by marcel-westhoff** [kimai/kimai#4423] [General]
Hello everybody,

is there any chance to sync my office 365 calendar like my outlook calendar?

Marcel

**Comment by mastertIT** [comment]
if kimai could utilize an ics file subscription that would be a universal way to sync any calendar into kimai, easy for a public shared calendar, if not public then one hurdle would be authentication, if it accepts basic auth then user@password:exchange.outlook.com/calandar/holidays.ics
otherwise separate auth classes would need to be built for every cloud provider / copy cookies?

@kevinpapst any thoughts on this? we would also use something similar, but via exchange.

**Reply by marcel-westhoff** [reply]
So I'm using exchange as well. How do you connect the exchange service? I have to use MFA within my accounts, because of governmental contracts.
